gccjit_sys/Cargo.toml

```
@@ -12,4 +12,4 @@ repository = "https://github.com/swgillespie/gccjit.rs"
 master = []
 
 [dependencies]
-libc = "0.1.6"
+libc = "0.2.140"
```

I'm not too familiar with semver, but I believe it would be more flexible to do the following:
```suggestion
libc = "0.2"
```
and use the version with s390x support in the project using this library, by having version `0.2.140` in the `Cargo.lock` of rustc_codegen_gcc, for instance.

---

This works for me as well.